polaris-react/src/components/Modal/Modal.stories.tsx

```diff
@@ -15,12 +16,16 @@ import {
   Frame,
   FrameContext,
 } from '@shopify/polaris';
+import {transitionsAllSettled} from 'transitions-all-settled';
 
 export default {
   component: Modal,
 } as Meta<typeof Modal>;
 
 export const Default = {
+  play: async ({canvasElement}) => {
+    await transitionsAllSettled(canvasElement);
```

Comment:
Prevents the accessibility checker from running while the CSS fade-in transition for Modal is still happening and giving false positives for color contrast issues.